src/Patches/InventoryItemSpec_Patch.cs

```diff
@@ -7,22 +7,22 @@ namespace custom_item_mod.Patches;
 /// <summary>
 /// IDK why but a NullReferenceException occurs without this patch. Pasted the exception at the bottom of this file.
 /// </summary>
-[HarmonyPatch(typeof(InventoryItemSpec))]
-[HarmonyPatch(nameof(InventoryItemSpec.LocalizedName), MethodType.Getter)]
-public class InventoryItemSpec_Patch
-{
-	private static Exception Finalizer(Exception __exception, ref InventoryItemSpec __instance, ref string __result)
-	{
-		if(__instance.itemPrefabName.StartsWith(Main.MyModEntry.Info.Id) &&
-		   __exception.GetType() == typeof(NullReferenceException))
-		{
-			__result = LocalizationAPI.L(__instance.localizationKeyName);
-			return null;
-		}
+//[HarmonyPatch(typeof(InventoryItemSpec))]
```

Comment:
looks like we dont need this patch anymore, lets delete it

Reply:
Done, I left that one in as I didn't fully understand it and didn't want to make the call to delete it myself.